webapi/middleware.go

```diff
@@ -25,8 +25,9 @@ type ticketHashRequest struct {
 }
 
 type ticketRequest struct {
-	TicketHex  string `json:"tickethex" binding:"required"`
-	TicketHash string `json:"tickethash" binding:"required"`
+	TicketHex  string  `json:"tickethex" binding:"required"`
+	TicketHash string  `json:"tickethash" binding:"required"`
+	ParentHex  *string `json:"parenthex,omitempty"`
```

Comment:
`omitempty` only effects marshalling, not unmarshalling, and thus it isn't needed on requests, only responses.

A field is made optional by omitting `binding:"required"`

Reply:
Also lets stay consistent with the rest of the fields and use a string here rather than a pointer. Otherwise it isn't immediately obvious whether the check for existence should be `!= nil`, `!= ""`, or both.